src/main/java/yablunovskyi/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,25 @@
+package yablunovskyi.bookstore.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.math.BigDecimal;
+import org.hibernate.validator.constraints.URL;
+
+public record CreateBookRequestDto(
+        @NotBlank
+        String title,
+        @NotBlank
+        String author,
+        @NotBlank
+        String isbn,
+        @NotNull
+        @Min(0)
+        BigDecimal price,
+        @Size(min = 10, max = 500)
+        String description,
+        @URL
```

Comment:
I don't think that @URL is matching for this field

Reply:
I believe that such a check can be (because what else, except a URL with a link to some picture, can be transferred to a field called coverImage)